src/storage.js

```diff
@@ -23,4 +26,41 @@ module.exports = {
   getFeeds: () => redis.smembers(FEEDS),
 
   getFeed: feedID => redis.hgetall(feedID),
+
+  //  addPost
+  addPost: async post => {
+    const postId = await redis.incr(POST_ID);
+    await redis
+      .multi()
+      .hmset(
+        postId,
+        'guid',
+        post.guid,
+        'author',
+        post.author,
+        'title',
+        post.title,
+        'link',
+        post.link,
+        'content',
+        post.content,
+        'text',
+        post.text,
+        'updated',
+        post.updated,
+        'published',
+        post.published,
+        'url',
+        post.url,
+        'site',
+        post.site
+      )
+      .zadd(POSTS, Date.parse(post.published), postId)
```

Comment:
I think that we can assume that `post.published` will already be a date vs. a date string.  Also, you can use `post.guid` instead of `postId`.

Reply:
@humphd In order to sort the set, the score value must be numeric. The Date.parse function will transfer the date string to an integer millisecond value based on how far ahead to a base time.

Reply:
Agree.  I'm saying, assume you have a `Date` and just call `getTime()` to get the `ms`:

```
> new Date()
2019-11-23T02:53:01.273Z
> new Date().getTime()
1574477586738
```

Reply:
OK, we can try this.